frappe/__init__.py

```diff
@@ -796,9 +796,9 @@ def get_doc(*args, **kwargs):
 
 	return doc
 
-def get_last_doc(doctype):
+def get_last_doc(doctype, **kwargs):
```

Comment:
Why not just pass filters? Can you think of any other parameter that will make sense passing in `get_last_doc`?

Reply:
I couldn't. That's why I added kwargs instead. I think I can add an override for `order_by` too, probably?

Reply:
@gavindsouza I think if the sort field is not `creation` then the method name doesn't make sense.

Reply:
It does if we use a different field to track the latest records created based on the real time associated with those records and not when they were added to my table.